app/src/main/java/com/github/neighbortrader/foodboardapp/clientmodel/Offer.java

```diff
@@ -79,11 +81,37 @@ public static Offer createOfferFromJSON(JSONObject jsonObject) {
         return null;
     }
 
+    public static Offer createRandomOffer() {
```

Comment:
vielleicht schreibst du lieber nen mock der in den Handler rein gegeben wird kenn ich zumindest so

Reply:
Kannst du das genauer erläutern?

Reply:
besprechen wir in der nächsten Pause

Reply:
ist bisschen komplexer